packages/generator-langium/langium-template/.vscode/tasks.json

```diff
@@ -0,0 +1,17 @@
+{
+    // See https://go.microsoft.com/fwlink/?LinkId=733558
+    // for the documentation about the tasks.json format
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "label": "Generate <%= language-id %>",
```

Comment:
This is more than just generating, it also involves the TypeScript compiler. Rename to "Build"?

Reply:
Done